evenodd.c

```diff
@@ -0,0 +1,18 @@
+/*Checks if user input is odd or even */
+#include <stdio.h>
+int main () {
+	int input;
+	printf("Enter a number:");
+	scanf("%d", &input);
+	if (input%2 == 0) {
+		printf("The number is even \n");
+	}
+	if (input%2 != 0) {
```

Comment:
Since this isn't an else if statement, "that's not even a number" is printed whenever a number is even. Nice try at trying to handle all input though!